homeassistant/components/binary_sensor/homematicip_cloud.py

```diff
@@ -86,3 +86,21 @@ def is_on(self):
         if self._device.sabotage:
             return True
         return self._device.motionDetected
+
+
+class HomematicipSmokeDetector(HomematicipGenericDevice, BinarySensorDevice):
+    """MomematicIP smoke detector."""
+
+    def __init__(self, home, device):
```

Comment:
This isn't needed if just passing through the arguments to the parent.

Reply:
Seems to be needed for multiple inheritance... if I remove it BinarySensorDevice is not initialised .

Reply:
`BinarySensorDevice` doesn't have an init method. What symptoms are you seeing?

Reply:
After clearing the build folder the symptoms are gone (I get again a "Clear" and not a False)